reana_job_controller/slurmcern_job_manager.py

```diff
@@ -121,7 +121,7 @@ def execute(self):
         self._dump_job_file()
         self._dump_job_submission_file()
         stdout = self.slurm_connection.exec_command(
-            "cd {} && sbatch --parsable {}".format(
+            "bash -c 'cd {} && sbatch --parsable {}'".format(
```

Comment:
I've initially added the fix in [`exec_command`](https://github.com/reanahub/reana-job-controller/blob/85db1afa28cf06f803a041df1b4f3ee31c0eb982/reana_job_controller/utils.py#L93) itself, but it doesn't play nicely with other quotes we have around the code (e.g. [here](https://github.com/reanahub/reana-job-controller/blob/85db1afa28cf06f803a041df1b4f3ee31c0eb982/reana_job_controller/job_monitor.py#L447)). Moreover, only this command is missing the correct environment, so I kept it minimal.